testing/py3-cymem/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=py3-cymem
+pkgver=2.0.2
+pkgrel=0
+pkgdesc="Cython memory pool for RAll-style memory management"
+url="https://github.com/explosion/cymem"
+arch="all"
+license="MIT"
+depends="python3"
+makedepends="python3-dev py3-setuptools cython3"
+checkdepends="py3-pytest py3-tox"
+subpackages="$pkgname-doc"
+source="cymem-$pkgver.tar.gz::https://github.com/explosion/cymem/archive/v$pkgver.tar.gz
+	tox.ini"
+builddir="$srcdir/cymem-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	cp "$srcdir"/tox.ini "$builddir"
+	tox -e py37
```

Comment:
why use tox at all and not just use call pytest directly ?

Reply:
I am unable to get tests running without error, I really miss something here even using `PYTHONPATH="$PWD/build/lib"`. I got tests running fine with this `tox` trick. Would quite appreciate any comments.